app/src/main/java/com/locoquest/app/dto/Photo.kt

```diff
@@ -3,8 +3,8 @@ package com.locoquest.app.dto
 import java.util.*
 
 data class Photo(
-    val localURL: String,
-    val remoteURL: String,
-    val description: String,
-    val benchmarkDate: Date,
-    val locationID: String)
+    var localURL: String,
+    var remoteURL: String,
+    var description: String,
+    var benchmarkDate: Date,
+    var locationID: String)
```

Comment:
Changed variable declaration from val to var. Var is general variable and can be assigned multiple times and is known as the mutable variable in Kotlin. Whereas val is a constant variable and can not be assigned multiple times and can be Initialized only single time.( https://developer.android.com/kotlin/learn#:~:text=Kotlin%20uses%20two%20different%20keywords,variable%20whose%20value%20can%20change.)

Reply:
In Kotlin, val is preferred over var.  It is safer to assume a variable is immutable, unless you have good reason to know it is mutable.